### ISSUE TITLE
Voice Attack & EDDI will CTD with FSD Drive

### ISSUE THREAD

**Issue by Nalsister** [cmdrmcdonald/EliteDangerousDataProvider#136]
It is a Google translation. I am sorry that I am not good at English.
I am using the EDDI plugin.
When entering EDDI Jumping with FSD Drive, Voice Attack will CTD.
It happens many times when CTD happens, but never happens when it does not happen.
My friend has no problem at all with the same setting, but what is the cause of the problem?
Please let me know how EDDI Jumping will not be CTD.

I use Windows10 Anniversary ver.

**Comment by cmdrmcdonald** [comment]
Please could you do the following:

  - remove %APPDATA%\EDDI\eddi.log
  - start EDDI standalone (not from VoiceAttack)
  - enable the 'Enable verbose logging' option on the first tab
  - jump (at this point you say EDDI crashes)
  - restart EDDI standalone
  - press the 'Send EDDI log to developers' button
  - respond on this thread to let me know

Thaks

**Comment by Viridol** [comment]
As I'm also experiencing this, I thought I'd step in and say that my log was just uploaded as per the instructions above. Cmdr Viridol if it is of any help finding it.

**Comment by cmdrmcdonald** [comment]
@Viridol Thank you for the log, however I don't see anything in there that suggests a jump from one system to another.  Please could you try again, and let me know which system you jumped to and from, and at what time?  Thank you.

**Comment by Viridol** [comment]
Oh, strange! All right, sorry for the delayed response. Submitted another and the jump was from LTT 17868 to Breksta.

I might also add the same crash is occuring frequently outside of jumping, but unpredictably as far as I can tell. A system jump is just a reliable way to reproduce it.

**Comment by cmdrmcdonald** [comment]
Apologies, but it looks like my instructions were incorrect.  If you could please do it again, but this time enable verbose logging *then* restart EDDI, then jump, it should catch the required information.

**Comment by Viridol** [comment]
Makes sense! I was wondering about how it didn't seem to generate a new eddi.log. :P

Right-o, guv. This time from Breksta to LTT 17868.

**Comment by cmdrmcdonald** [comment]
There's nothing in the log that suggests why the crash occurs, which in itself is odd.  The best I can come up with is that it's totally outside of EDDI, which points to the speech synthesis layer.

Do you have a different voice you can try?  I have heard of other users crashing with Salli, and was never able to track it down, but it might be a bug in the voice itself.

**Comment by Viridol** [comment]
Huh. Yes, Microsoft Zira and David perform fine, while my Ivona voices (Salli, Amy, and Ivy) all exhibit this crash.

What's especially weird about that to me is that there were no crashes the first day using Salli, at all, with over 40 jumps. This got me thinking about just reinstalling the Ivona packs, but no dice there. Curious...

Anyway, thank you for the help!

**Comment by Viridol** [comment]
Ah! So, interesting find:

Checking "Disable Phonetic Speech" in EDDI appears to make the IVONA voices happy, though it sure does cripple their excellent pronunciation. Still, I was considering it very strange that it seemed fine using phonetic speech it for so long before, and...

Turns out that it's apparently all about the ship I'm flying! I remembered that I switched from an Adder to a Cobra Mk. III the other day, and so by switching around the "Ship for testing voice" option in EDDI, I found that as long as "Disable phonetic speech" is _not_ checked, it crashes when asked to test anything with "Mk." or the Fer-de-Lance.

Adding these terms to its lexicon doesn't seem helpful, though, so I'm inclined to just go into the EDDI personality and replace all the {ShipName()} with "ship" or something.

**Comment by cmdrmcdonald** [comment]
The way that phonetic speech is handled has been altered in EDDI 2.3 (currently in beta).  Please give it a try and see if this fixes your issue.
